apps/events-helsinki/src/pages/pages/[...slug].tsx

```diff
@@ -102,7 +99,6 @@ type ResultProps =
   | {
       initialApolloState: NormalizedCacheObject;
       page: PageQuery['page'];
-      breadcrumbs: Breadcrumb[];
```

Comment:
I would have kept this field in the type.

Reply:
Made a new commit that returns this.